docs/manage/connections-integrations/webhook-connections/set-up-webhook-connections.md

```diff
@@ -50,10 +52,11 @@ To set up a webhook connection:
     :::
 1. (Optional) If the third-party system requires an **Authorization Header**, enter it here. For more information, see [Example Authorization Header](#example-authorization-header) below.
 1. (Optional) **Custom Headers**, enter up to five comma separated key-value pairs.
-1. For **Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of either `Critical` , `Warning`, or `Missing ``Data`. Recovery will have either `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. The same payload for both alerts and recovery is used.
+1. Under **Alert Payload**, enter a JSON object in the format required by the target webhook URL. For details on variables that can be used as parameters within your JSON object, see [webhook payload variables](#webhook-payload-variables), below. When using a generic webhook connection for notifications in a [Monitor](/docs/alerts/monitors) you can distinguish between an alert and a recovery by using the [alert variable](../../../alerts/alert-variables.md) `TriggerType`. An alert will have a value of `Critical`, `Warning`, or `Missing Data`. Recovery will have `CriticalResolved`, `WarningResolved`, or `Missing Data Resolved`. <br/><img src={useBaseUrl('img/connection-and-integration/alert-payload.png')} alt="recovery payload" />
```

Comment:
"Alert Payload" 

Here we should describe in a line or two what is alert payload vs recovery payload. 

Alert Payload allows users to customize how the alert notification will look like. 

Recovery Payalod allows users to customize how the Recovery notification for an alert looks like.